tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -449,10 +449,11 @@ def test_parsing_context(self):
             "_AIRFLOW_PARSING_CONTEXT_TASK_ID=task1\n"
         )
 
+    @pytest.mark.db_test
     @mock.patch("airflow.task.task_runner.standard_task_runner.Stats.gauge")
     @patch("airflow.utils.log.file_task_handler.FileTaskHandler._init_file")
-    def test_read_task_utilization(self, mock_init, mock_stats):
-        mock_init.return_value = "/tmp/any"
+    def test_read_task_utilization(self, mock_init, mock_stats, tmp_path):
+        mock_init.return_value = (tmp_path / "test_read_task_utilization.log").as_posix()
```

Comment:
That is the minor change, for change hardcoded `/tmp/any` to `tmp_path` path fixture which use separate location for each run, and also auto removable in case of success test (our configuration for lifecycle)

I have no idea how to prevent to use "/tmp/..." it very easy to miss it during review 😞